pkg/collector/udp.go

```diff
@@ -42,12 +43,16 @@ func (cp *collectingProcess) startUDPServer() {
 		for {
 			buff := make([]byte, cp.maxBufferSize)
 			size, address, err := conn.ReadFromUDP(buff)
-			cp.handleUDPClient(address, &wg)
-			cp.clients[address.String()].packetChan <- bytes.NewBuffer(buff[0:size])
 			if err != nil {
+				if size == 0 { // received stop collector message
+					return
+				}
 				klog.Errorf("Error in collecting process: %v", err)
 				return
 			}
+			klog.Infof("Receiving %d bytes from %s", size, address.String())
```

Comment:
Here as well. Having a log with level 0, whenever new client is added would be good and rest should be either 2 or 4. What do you think?

Reply:
Yes I agreed. Log level 2 is better than 0 here